cmd/main.go

```diff
@@ -0,0 +1 @@
+package cmd
```

Comment:
The path should be /cmd/game/main.go for the future executable "game" - just community convention. Package will be "main" for this file.